src/modules/Notifies/X11_Notify.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+	QMPlay2 is a video and audio player.
+	Copyright (C) 2010-2016  Błażej Szczygieł
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Lesser General Public License as published
+	by the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Lesser General Public License for more details.
+
+	You should have received a copy of the GNU Lesser General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <Notify.hpp>
+
+#include <QDateTime>
+
+class OrgFreedesktopNotificationsInterface;
+class QDBusPendingCallWatcher;
+
+class X11_Notify : public Notify
```

Comment:
Maybe `X11Notify`?
I prefer using camelCase instead of "_". I know that QMPlay2 has mixed styles, but new code has always `camelCase` for variables, `CamelCase` for classes name and `m_camelCase` for private/protected class members.
It can be applied also for INI (settings) keys (start e.g. from capital letter).